src/form/time/dropdown/TimeDropdown.tsx

```diff
@@ -0,0 +1,79 @@
+import "./_time-dropdown.scss";
+
+import React, {useEffect, useState} from "react";
+import classNames from "classnames";
+
+import Dropdown from "../../../dropdown/Dropdown";
+import {DropdownOption} from "../../../dropdown/list/item/DropdownListItem";
+import {generateTimeDropdownOptions} from "./util/timeDropdownUtils";
+
+export type TimeDropdownOption = DropdownOption<
+  string,
+  {date: Date; differenceInMinutes: number}
+>;
+
+export interface TimeDropdownProps {
+  testid: string;
+  startTimeString: string;
```

Comment:
I think we need to optionally expect a `startDate`  as well. Otherwise, TimeDropdown will always have the options within today, https://github.com/Hipo/react-ui-toolkit/pull/83/files#diff-8852021072415203fa35795a478d99809c89a9f47c3ed888013a275b44916bbaR65.